docs/installation-configuration.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Defang Provider for Pulumi Installation & Configuration
+meta_desc: Provides an overview on how to configure the Pulumi Defang Provider.
+layout: package
+---
+## Installation
+
+The Pulumi provider for [Defang] (https://defang.io) - a radically simpler way to develop, deploy, and debug cloud applications. The easiest way to deploy your Docker Compose project to the cloud with Pulumi.
+
+The Defang Pulumi provider, is available in most pulumi languages. 
+
+* JavaScript/TypeScript: [`@defang-io/pulumi-defang`](https://www.npmjs.com/package/@defang-io/pulumi-defang)
+* Python: [`pulumi-defang`](https://pypi.org/project/pulumi-defang/)
+* Go: [`github.com/DefangLabs/pulumi-defang/sdk/v1/go/defang`](https://github.com/DefangLabs/pulumi-defang)
+* Java: Coming soon
+* Dotnet: Coming soon
+
+## Authentication
+
+Sign up for [Defang](https://defang.io) with your Github account. When run in a Github Actions workflow, the Defang Pulumi provider will automatically use environment varialbes Github providew to authenticate your Github user with Defang if you give your workflow the [appropriate permissions](https://docs.github.com/en/actions/security-for-github-actions/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings). Defang use the `ACTIONS_ID_TOKEN_REQUEST_URL` and `ACTIONS_ID_TOKEN_REQUEST_TOKEN` env vars.
```

Comment:
We also support `DEFANG_ACCESS_TOKEN` (from `defang token`) as an alternative to the id_token auth.